go.sum

```diff
@@ -270,6 +270,8 @@ github.com/lestrrat-go/iter v1.0.1 h1:q8faalr2dY6o8bV45uwrxq12bRa1ezKrB6oM9FUgN4
 github.com/lestrrat-go/iter v1.0.1/go.mod h1:zIdgO1mRKhn8l9vrZJZz9TUMMFbQbLeTsbqPDrJ/OJc=
 github.com/lestrrat-go/jwx v1.2.12-0.20211125230033-881a53637884 h1:3o0vWB6NHNhgtxVwQdgFvqGFf8BWyS/urvmLONVsUeI=
 github.com/lestrrat-go/jwx v1.2.12-0.20211125230033-881a53637884/go.mod h1:25DcLbNWArPA/Ew5CcBmewl32cJKxOk5cbepBsIJFzw=
```

Comment:
These `v1.2.12-0.20211125230033-881a53637884` lines are removed from `go.sum` when I run `go mod tidy`.

Reply:
yeah, that's my bad, I shoulda waited for the final module version before opening that PR. Pretty sure you just gotta delete these two lines like zrhoffman says, tho

Reply:
weird... I wonder why I can't just pick these up by running go mod tidy in my environment.

Reply:
should be removed now!